src/main/java/com/unstoppabledomains/resolution/CNS.java

```diff
@@ -0,0 +1,125 @@
+
+package com.unstoppabledomains.resolution;
+
+import java.math.BigInteger;
+import java.net.UnknownHostException;
+
+
+import org.web3j.tx.exceptions.ContractCallException;
+
+import com.unstoppabledomains.exceptions.NSExceptionCode;
+import com.unstoppabledomains.exceptions.NSExceptionParams;
+import com.unstoppabledomains.exceptions.NamingServiceException;
+import com.unstoppabledomains.resolution.contracts.cns.Registry;
+import com.unstoppabledomains.resolution.contracts.cns.Resolver;
+
+public class CNS extends NamingService {
```

Comment:
Java support `protected` class to make sure class is only visible inside the package. I think we should make it such.

Reply:
Illegal modifier for the class CNS; only public, abstract & final are permitted

This is what I got whenever I change the CNS to be protected